tests/unit/services/SearchTest.php

```diff
@@ -289,7 +289,7 @@ private function _scoreList(array $usernameOrEmailsAndScores): array
      * @param string $emailOrUsername
      * @return User|null
      */
-    private function _getUserIdByEmailOrUserName(string $emailOrUsername): ?User
```

Comment:
Sorry! Internally we are sticking to 7.3 now for everything so I got a bit carried away.